pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -44,14 +45,17 @@ const (
 
 // Fit is a plugin that checks if a node has sufficient resources.
 type Fit struct {
-	ignoredResources sets.String
+	ignoredResources      sets.String
+	ignoredResourceGroups sets.String
 }
 
 // FitArgs holds the args that are used to configure the plugin.
 type FitArgs struct {
 	// IgnoredResources is the list of resources that NodeResources fit filter
 	// should ignore.
 	IgnoredResources []string `json:"ignoredResources,omitempty"`
+	// IgnoredResourceGroups defines the list of resource prefix that NodeResources fit filter should ignore.
```

Comment:
group instead of prefix. Don't be afraid of redundancy :)

Also, an example would be useful.